search/templates/search/search-macros.html

```diff
@@ -129,26 +129,84 @@
+            <span class="tag is-light is-size-7"><a class="" href="https://dx.doi.org/{{ doi }}">{{ doi }} <i class="fa fa-external-link" aria-hidden="true"></i></a></span>
+          </div>
+        </div>
+        {% endfor %}
+      </div>
+      {% endif %}
+    </div>
     {# Note: mathjax class should be applied to any element that requires MathJax processing #}
-    <p class="title is-5 mathjax">{{ result.title_utf8 }}</p>
-    <p class="authors">Authors:
+    <p class="title is-5 mathjax">
+
+      {% if result.highlight.title_utf8 %}
+        {{ result.highlight.title_utf8 | safe }}
+      {% else %}
+        {% if result.highlight.title %}
+          {{ result.highlight.title | safe }}
+        {% else %}
+          {{ result.title_utf8 }}
+        {% endif %}
+      {% endif %}
+    </p>
+    <p class="authors is-size-7">
+      <span class="has-text-black-bis has-text-weight-semibold">Authors:</span>
       {% for author in result.authors[0:25] %}
       <a href="{{ url_for_author_search(author.first_name, author.last_name) }}">{{ author.first_name }} {{ author.last_name }}{% if author.suffix %} {{ author.suffix }}{%- endif -%}</a>{{ ", " if not loop.last }}
       {% endfor -%}
       {% if result.authors | length > 25 %}, et al. ({{ result.authors | length - 25 }} additional authors not shown){% endif %}
     </p>
-    <p>Submitted {{ result.submitted_date.strftime('%-d %B, %Y at %H:%M:%S %Z') }}{% if result.version > 1 %}; <a class="has-text-weight-bold" href="{{ external_url('browse', 'abstract', paper_id=result.paper_id) }}v1">v1</a> submitted {{ result.submitted_date_first.strftime('%-d %B, %Y at %H:%M:%S %Z') }}{% endif %}. {% if result.latest %}<span class="has-text-weight-bold">Latest version: <a href="{{ external_url('browse', 'abstract', paper_id=result.latest) }}">{{ result.latest }}</a></span>{% endif %}</p>
-    {% if result.comments %}<p class="comments">Comments: {{ result.comments }}</p>{% endif %}
+    <p class="abstract mathjax  is-size-7">
+      <span class="has-text-black-bis has-text-weight-semibold">Abstract</span>:
+        <span class="abstract-short has-text-grey-dark" id="{{result.id}}-abstract-short" style="display: inline;">
+          {% if result.highlight.abstract %}{{ result.highlight.abstract | safe }}{% else %}{{ result.abstract | truncate(175) }}{% endif %}
+          <a class="is-size-7" onclick="document.getElementById('{{result.id}}-abstract-full').style.display = 'inline'; document.getElementById('{{result.id}}-abstract-short').style.display = 'none';">&#9661; More</a>
+        </span>
+        <span class="abstract-full has-text-grey-dark" id="{{result.id}}-abstract-full" style="display: none;">
+          {{ result.abstract }}
+          <a class="is-size-7" onclick="document.getElementById('{{result.id}}-abstract-full').style.display = 'none'; document.getElementById('{{result.id}}-abstract-short').style.display = 'inline';">&#9651; Less</a>
+        </span>
+    </p>
```

Comment:
Unfortunately MathJax rendering didn't work for me in the abstracts. I think you'll have to move the mathjax class to the spans with `abstract-short` and `abstract-long`, because the config basically disallows inheritance.

Reply:
Done; does that fix it?

Reply:
Ah, wait, I see the problem now... one moment.